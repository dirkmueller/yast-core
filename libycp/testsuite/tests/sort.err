Parsed:
----------------------------------------------------------------------
sort (`x, `y, [["Advanced-Devel", " Advanced Development", "addon"], ["Devel", " Development", "addon"], ["Games", " Games", "addon"], ["Gnome", " Gnome system", "addon"], ["Kde", " All of KDE", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Linux-Devel", " Linux Development Tools", "addon"], ["Minimal", " Minimum system", "baseconf"], ["Minimal+X11", " Minimum graphical system (without KDE)", "baseconf"], ["Multimedia", " Multimedia", "addon"], ["Network", " Network/Server", "addon"], ["Office", " Office Applications", "addon"], ["Standard", " Default system", "baseconf"], ["SuSE-Documentation", " Help & Support Documentation", "addon"], ["X11", " X11", "addon"], ["default", " Default system", "baseconf"], ["hilbert", " Hilbert/Machcd Server", "baseconf"], ["zzz_Alles", " All packages (very time-consuming)", "baseconf"]], ``((x[1]:"" <= y[1]:"")))
----------------------------------------------------------------------
