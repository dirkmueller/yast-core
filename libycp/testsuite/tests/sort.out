([["Advanced-Devel", " Advanced Development", "addon"], ["Kde", " All of KDE", "addon"], ["zzz_Alles", " All packages (very time-consuming)", "baseconf"], ["Standard", " Default system", "baseconf"], ["default", " Default system", "baseconf"], ["Devel", " Development", "addon"], ["Games", " Games", "addon"], ["Gnome", " Gnome system", "addon"], ["SuSE-Documentation", " Help & Support Documentation", "addon"], ["hilbert", " Hilbert/Machcd Server", "baseconf"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["Linux-Devel", " Linux Development Tools", "addon"], ["Minimal+X11", " Minimum graphical system (without KDE)", "baseconf"], ["Minimal", " Minimum system", "baseconf"], ["Multimedia", " Multimedia", "addon"], ["Network", " Network/Server", "addon"], ["Office", " Office Applications", "addon"], ["LAMP", " Simple Webserver", "addon"], ["X11", " X11", "addon"]])
